src/clj/game/cards/resources.clj

```diff
@@ -1138,7 +1138,9 @@
     :abilities [{:cost [:credit 2] :msg "avoid 1 tag" :effect (effect (tag-prevent 1))}]}
 
    "No One Home"
-   (letfn [(start-trace [type]
+   (letfn [(first-chance? [state side] (and (< (event-count state side :pre-tag) 2)
+                                            (< (event-count state side :pre-damage) 2)))
```

Comment:
Should this be "the sum of pre-tag and pre-damage is less than 2"? Since it's first time for either of those events.

Reply:
Good catch - I think you have that right.  I spent a while banging my head on this one due to the pre-damage firing before the prevent.

* First damage fires and we get :pre-damage event which other cards use. So count of pre-damage is now 1.
* Then the prevention logic fires and we check in the No One Home req that the count of pre-damage events is less than 2 (ie. this is the first time).  Strictly I could check for exactly 1 but want with less than 2.

Now let's assume I get my first tag event after this
* First tag event happens and we get a :pre-tag event fire.  Count of :pre-tag = 1, count of :pre-damage
* No One Home will not pass the req check

OR
* I get my second damage effect
* Count of :pre-damage = 2 so NOH will not pass.